### PR TITLE
Using array made a arraylist and tested .

### PR DESCRIPTION
**Pull request by sreeragps**
using array and made a arraylist.

### CODE REVIEW

**Review by aneeshs005 — COMMENTED** [review]
LGTM

File: C#/cs01/15.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+namespace Brainchild.Curriculum.Solutions
+{
+    // ArrayList like class IntList
+    // IntList should have two methods
+    // Add(int num)
+    // Get(int index)
+    // Count Property
+    public class IntList
+    {
+        public int[] integerArray = new int[1];
```
Comment (rohithkrajan) [inline C#/cs01/15.cs]: variable names should not be reflecting C# names like int or float etc. you can name it like internalList or InternalArr. something which makes sense

**Review by rohithkrajan — CHANGES_REQUESTED** [review]
just a small suggestion otherwise it looks good

File: C#/cs01/15.cs (hunk 1 above)
Comment (sreeragps) [reply C#/cs01/15.cs]: sure ,here on I will name it like that

**Review by aneeshs005 — APPROVED** [review]
LGTM

**Review by rohithkrajan — APPROVED** [review]

**Pull request merged**